### ISSUE TITLE
VMagent: not follow updates in the `global` section on reload

### ISSUE THREAD

**Issue by Vladikamira** [VictoriaMetrics/VictoriaMetrics#2884]
**Describe the bug**
Vmagent does not apply changes in the `global` section of config, but it does for `scrape_configs`, here is an example of config:
```
    global:
      external_labels:
        test_label: some_text2
      scrape_interval: 10s

    scrape_configs:
    - job_name: kubernetes-nodes
      kubernetes_sd_configs:
      - role: node
...
```
we've configured config by this flag `-promscrape.config=/config/scrape.yml` and check by this one `-promscrape.configCheckInterval`

**To Reproduce**
add some label into `extra_labels` section and trigger reload

**Expected behavior**
On file change vmagent should reload and apply new changes without restart

**Version**
The line returned when passing `--version` command line flag to the binary. For example:
```
/ # /vmagent-prod --version
vmagent-20220714-135650-tags-v1.79.0-0-gd3116d986
```

**Used command-line flags**
```
    Args:
      -promscrape.config=/config/scrape.yml
      -remoteWrite.tmpDataPath=/tmpData
      -remoteWrite.url=http://victoria-metrics-cluster-a-vminsert.monitoring.svc.cluster.local:8480/insert/0/prometheus/
      -remoteWrite.url=http://victoria-metrics-cluster-b-vminsert.monitoring.svc.cluster.local:8480/insert/0/prometheus/
      -envflag.enable=true
      -envflag.prefix=VM_
      -http.shutdownDelay=30s
      -loggerErrorsPerSecondLimit=2
      -loggerFormat=json
      -loggerLevel=WARN
      -loggerOutput=stdout
      -loggerWarnsPerSecondLimit=1
      -memory.allowedPercent=60
      -promscrape.config.strictParse=false
      -promscrape.configCheckInterval=10s
      -promscrape.maxScrapeSize=1024MB
      -promscrape.suppressScrapeErrors=true
      -remoteWrite.maxBlockSize=16MB
      -remoteWrite.maxRowsPerBlock=20000
      -remoteWrite.showURL=true
```

**Comment by valyala** [comment]
The issue should be fixed in the commit 0fd86e23640f6c8b1bc94f24becd9417fdd123ea . This commit will be included in the next release. In the mean time you can build vmagent from this commit according to [these docs](https://docs.victoriametrics.com/vmagent.html#how-to-build-from-sources)

**Comment by valyala** [comment]
FYI, the bugfix has been included in [vmagent v1.79.2](https://github.com/VictoriaMetrics/VictoriaMetrics/releases).
